feelslikemonday/app/src/main/java/com/example/feelslikemonday/ui/friends/MyFriendsViewModel.java

```diff
@@ -0,0 +1,27 @@
+package com.example.feelslikemonday.ui.friends;
+
+import androidx.lifecycle.LiveData;
+import androidx.lifecycle.MutableLiveData;
+import androidx.lifecycle.ViewModel;
+
+/*This class is responsible for preparing and managing the data for FollowerRequest*/
+public class MyFriendsViewModel extends ViewModel {
```

Comment:
Are we sure we need classes like these? I'm not familiar with the Android UI stuff

Reply:
idk.  Ning puts at the beginning version so I keep for every fragments